includes/options/views/class-amp-analytics-options-serializer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+class Analytics_Options_Serializer {
+
+	public static function save() {
```

Comment:
(1) Good idea re:Thinking about the future. The options are saved now according to the hierarchy you suggested. 
(2) Thinking about using the Settings API. Need to change everything? :(